Mapping/Tests/Functional/Mapping/Reader/SearchMappingReaderTest.php

```diff
@@ -21,6 +21,9 @@ public function setUp()
         $this->fakeClass = new FakeClassAnnotation();
 
         static::bootKernel();
+        if (!static::$kernel->getContainer()->hasParameter('open_orchestra.annotation_search_reader')) {
+            $this->markTestSkipped('no service annotation reader');
```

Comment:
If the setting is present, you should check its value (media app should set it to false)